### DISCUSSION TITLE
PGMail Multiple Recipients

### DISCUSSION THREAD

**Discussion by Julangel03** [captbrando/pgMail#12] [General]
Hello, 

How do you add multiple recipients for the pgMail? 
Do you use any separators? Because I test using a single email address and it just works fine, but adding multiple recipients the email does not sent.

Thank you

**Comment by captbrando** [comment]
At this point, pgMail is not designed for multiple recipients. Generally, triggers in databases were envisioned to be coming from single record edits (which yes, could be in bulk, but each email will be run separately). Is there a specific use case that I should consider?

**Reply by Julangel03** [reply]
Hello

Thank you for the feedback, we are planning to have  the email sent via pgmail to the whole team who will need to do or support the activity in our database.

We are considering just looping the function to send to each team member as what you also mention that will be run separately. 🙂 


Thank you!

**Reply by captbrando** [reply]
That could work, but if you can get a group email alias that might help as well! 

Best, and thanks for the note!
